### PR TITLE
Widespace Bid Adapter: Support Adomain for prebid 5.0

### PR DESCRIPTION
**Pull request by talha08**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- Adomain support 
```
  meta: {
      advertiserDomains: bid.adomain || []
  }
```

### CODE REVIEW

File: modules/widespaceBidAdapter.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Overview
 
 
-**Module Name:** Widespace Bidder Adapter.
-**Module Type:** Bidder Adapter.
+**Module Name:** Widespace Bidder Adapter.\
```
[the remark this answers is not in the document]
Comment (talha08) [reply modules/widespaceBidAdapter.md:4]: Yes, For Looking good.

File: modules/widespaceBidAdapter.js [hunk 2]
```diff
@@ -95,10 +95,9 @@ export const spec = {
 
       // Include debug data when available
       if (!isInHostileIframe) {
-        const DEBUG_AD = (find(window.top.location.hash.split('&'),
+        data.forceAdId = (find(window.top.location.hash.split('&'),
```
Comment (patmmccann) [inline modules/widespaceBidAdapter.js:98]: please also fix line 64 above to solve #6465

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/widespaceBidAdapter.js (hunk 2 above)
Comment (patmmccann) [reply modules/widespaceBidAdapter.js:98]: feel free to do this on a different pr

**Comment by patmmccann** [comment]
It appears circleci did not run, this is usually because you are following your own project.

**Comment by patmmccann** [comment]
I got tests to run

File: modules/widespaceBidAdapter.js (hunk 2 above)
Comment (talha08) [reply modules/widespaceBidAdapter.js:98]: So, should I include this code with the new PR, or just fix the bidfloor issue there?

Comment (patmmccann) [reply modules/widespaceBidAdapter.js:98]: your choice

**Comment by patmmccann** [comment]
why did you close, this was ready to merge, tests passed

**Comment by patmmccann** [comment]
LGTM

**Pull request merged**